versions/1.1/SPEC.md

```diff
@@ -0,0 +1,5775 @@
+  - [✨ Array[String] quote(Array[P])](#-arraystring-quotearrayp)
+  - [✨ Array[String] squote(Array[X])](#-arraystring-squotearrayx)
+  - [✨ String sep(String, Array[String])](#-string-sepstring-arraystring)
+  - [✨ Array[Pair[P, Y]] as_pairs(Map[P, Y])](#-arraypairp-y-as_pairsmapp-y)
+  - [✨ Map[P, Y] as_map(Array[Pair[P, Y]])](#-mapp-y-as_maparraypairp-y)
+  - [✨ Array[P] keys(Map[P, Y])](#-arrayp-keysmapp-y)
+  - [✨ Map[P, Array[Y]] collect_by_key(Array[Pair[P, Y]])](#-mapp-arrayy-collect_by_keyarraypairp-y)
+  - [Boolean defined(X?)](#boolean-definedx)
+  - [X select_first(Array[X?]+)](#x-select_firstarrayx)
+  - [Array[X] select_all(Array[X?])](#arrayx-select_allarrayx)
+- [Input and Output Formats](#input-and-output-formats)
+  - [JSON Input Format](#json-input-format)
+    - [Optional Inputs](#optional-inputs)
+  - [JSON Output Format](#json-output-format)
+  - [Specifying / Overriding Runtime Attributes](#specifying--overriding-runtime-attributes)
+  - [JSON Serialization of WDL Types](#json-serialization-of-wdl-types)
+    - [Primitive Types](#primitive-types-1)
+    - [Array](#array)
+    - [Struct and Object](#struct-and-object)
+    - [Pair](#pair)
+    - [Map](#map)
+- [Appendix A: WDL Value Serialization and Deserialization](#appendix-a-wdl-value-serialization-and-deserialization)
+  - [Primitive Values](#primitive-values)
+  - [Compound Values](#compound-values)
+    - [Array](#array-1)
+      - [Array serialization by expansion](#array-serialization-by-expansion)
+      - [Array serialization using write_lines()](#array-serialization-using-write_lines)
+      - [Array serialization using write_json()](#array-serialization-using-write_json)
+      - [Array deserialization using read_lines()](#array-deserialization-using-read_lines)
+      - [Array deserialization using read_json()](#array-deserialization-using-read_json)
+    - [Struct and Object](#struct-and-object-1)
+      - [Struct serialization using write_json()](#struct-serialization-using-write_json)
+    - [Map](#map-1)
+      - [Map serialization using write_map()](#map-serialization-using-write_map)
+      - [Map serialization using write_json()](#map-serialization-using-write_json)
+      - [Map deserialization using read_map()](#map-deserialization-using-read_map)
+      - [Map deserialization using read_json()](#map-deserialization-using-read_json)
+    - [Pair](#pair-1)
+- [Appendix B: WDL Namespaces and Scopes](#appendix-b-wdl-namespaces-and-scopes)
+  - [Namespaces](#namespaces)
+  - [Scopes](#scopes)
+    - [Global Scope](#global-scope)
+    - [Task Scope](#task-scope)
+    - [Workflow Scope](#workflow-scope)
+    - [Cyclic References](#cyclic-references)
+    - [Namespaces without Scope](#namespaces-without-scope)
+  - [Evaluation Order](#evaluation-order)
+
+## Introduction
+
+WDL is meant to be a *human readable and writable* way to express tasks and workflows. Here is the "Hello World" tool in WDL:
+
```

Comment:
Does someone have an actual diagram of the essential parts of a WDL workflow that would be useful here to orient someone new to the spec?  Starting off with an example written IN WDL assumes that means something to the person reading it.  But if they're new to the spec and reading the intro still, they likely aren't.  No masterpiece required, but surely someone has a handy portion of a slide they've used in presentations or something?

Reply:
That's a good suggestion - I don't have anything but maybe @vdauwera does?